source/index.html.md

```diff
@@ -322,6 +322,10 @@ error categories, the last two digits define specific errors.
 * 1202: We cannot send an SMS to the number provided (out of jurisdiction)
 * 1299: Unexpected error when trying to send SMS
 
+### Reserved for internal use: 13 (425 error)
```

Comment:
Do we even need this here?
If there was other 425 errors with codes that an API user would see then yes, otherwise just get rid of it

Reply:
removed